### PR TITLE
Refactor FXIOS-6340 [v119] Replaced DeviceFontExtraLarge with preferredFont #14260

### PR DESCRIPTION
**Pull request by akshat3358**
## :scroll: Tickets
[Jira ticket](https://mozilla-hub.atlassian.net/browse/FXIOS-6340)
[Github issue](https://github.com/mozilla-mobile/firefox-ios/issues/14260)

## :pencil: Checklist
You have to check all boxes before merging
- [x] Filled in the above information (tickets numbers and description of your work)
- [x] Updated the PR name to follow our [PR naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [ ] Wrote unit tests and/or ensured the tests suite is passing
- [ ] When working on UI, I checked and implemented accessibility (minimum Dynamic Text and VoiceOver)
- [ ] If needed I updated documentation / comments for complex code and public methods

### CODE REVIEW

**Comment by akshat3358** [comment]
@isabelrios , @lmarceau  can you please review this ?

**Comment by akshat3358** [comment]
@isabelrios , @lmarceau  can you please review this ?

File: Client/Frontend/PasswordManagement/DevicePasscodeRequiredViewController.swift [hunk 1]
```diff
@@ -9,7 +9,7 @@ class DevicePasscodeRequiredViewController: SettingsViewController {
     private var warningLabel: UILabel = {
         let label = UILabel()
         label.translatesAutoresizingMaskIntoConstraints = false
-        label.font = LegacyDynamicFontHelper().DeviceFontExtraLarge
+        label.font = DefaultDynamicFontHelper.preferredFont(withTextStyle: .body, size: 17)
```
Comment (thatswinnie) [inline Client/Frontend/PasswordManagement/DevicePasscodeRequiredViewController.swift:12]: Looks like I wrote the wrong font size in the ticket and it should actually be 19. Sorry about that! Could you please change it everywhere?

**Review by thatswinnie — COMMENTED** [review]
Currently this branch isn't building because `import Common` is missing for both classes.

File: Client/Frontend/PasswordManagement/DevicePasscodeRequiredViewController.swift (hunk 1 above)
Comment (akshat3358) [reply Client/Frontend/PasswordManagement/DevicePasscodeRequiredViewController.swift:12]: @thatswinnie  , Are there any other changes required other than font size ?

Comment (thatswinnie) [reply Client/Frontend/PasswordManagement/DevicePasscodeRequiredViewController.swift:12]: @akshat3358 thanks for making this change. The only other change necessary would be to add `import Common` to `DevicePasscodeRequiredViewController.swift` and `PasswordManagerOnboardingViewController.swift` like mentioned above.

**Review by thatswinnie — APPROVED** [review]

**Pull request merged**